streams/src/test/java/org/apache/kafka/streams/kstream/internals/SuppressScenarioTest.java

```diff
@@ -198,11 +196,9 @@ public void shouldSuppressIntermediateEventsWithTimeLimit() {
                     new KeyValueTimestamp<>("v1", 1L, 2L)
                 )
             );
-            // note that the current stream time is 2, which causes v1 to age out of the buffer, since
-            // it has been buffered since time 0 (even though the current version of it in the buffer has timestamp 1)
```

Comment:
Why remove this comment? Seems to be valid?

Reply:
It's not anymore. Now, we buffer the new event before we enforce the buffer constraints, so we return the more intuitive most recent state of `"v1", 1L, 2L` right away, instead of later on.

Reply:
I see. The comment focus on the second `v1` -- I applied it to the third `v1`. Seems to comment was ambiguous :)

Reply:
Good thing it's gone!